helpers/for.js

```diff
@@ -3,8 +3,8 @@
 const _ = require('lodash');
 
 const factory = () => {
-    return function(from, to, context) {
-        const options = arguments[arguments.length - 1];
+    return function(from, to, context, options) {
+        options =  options ? options : context ? context : to;
```

Comment:
Nested ternary operators should probably be disambiguated with parentheses. Even if Javascript knows what to do here, it's confusing for the reader.

Reply:
will do.